src/vellum/workflows/expressions/greater_than.py

```diff
@@ -35,7 +35,14 @@ def resolve(self, state: "BaseState") -> bool:
         lhs = resolve_value(self._lhs, state)
         rhs = resolve_value(self._rhs, state)
 
-        if not has_gt(lhs):
-            raise InvalidExpressionException(f"'{lhs.__class__.__name__}' must support the '>' operator")
+        if not isinstance(lhs, (int, float)):
```

Comment:
this doesn't look right. please revert these impl's and let's start with just the test for now